src/bamber19_icesheets/bamber19_icesheets_project.py

```diff
@@ -0,0 +1,459 @@
+            "AIS", ais_samps, years, samples, locations, scenario, baseyear
+        )
+        ds_ais.to_netcdf(output_AIS_gslr_file)
+
+    if output_GIS_gslr_file is not None:
+         
+        ds_gis = make_projection_ds(
+            "GIS", gis_samps, years, samples, locations, scenario, baseyear
+        )
+        ds_gis.to_netcdf(output_GIS_gslr_file)
+
+    return icesheets_output  
+
+
+def GetSATData(
+    climate_data_file,  # NOTE was fname
+    scenario,
+    refyear_start=1850,
+    refyear_end=1900,
+    year_start=1900,
+    year_end=2300,
+):
+    """ "
+    This function takes a filename (climate_data_file) and other args. Reads an hdf5 file that has surface_temperature for different scenarios.
+    - Looks at number of ensemble members for that scenario,
+    - Extracts the years available in the data,
+    - Trims to range of years requested in inputs,
+    - Normalize, crop temp series,
+    - Return normalized SAT (Surface air temp) & time arrays and number of ensemble members.
+    """
+
+    df_ssp = h5py.File(climate_data_file, "r")
+
+    # Extract surface temperature for specified scenario
+    if scenario not in df_ssp.keys():
+        raise ValueError(
+            "Scenario {} not found in climate data file {}".format(
+                scenario, climate_data_file
+            )
+        )
+    ssp_folder = df_ssp.get(scenario)
+    sat_ssp = ssp_folder.get("surface_temperature")
+
+    # Get number of ensemble members from the data
+    _, nens = sat_ssp.shape
+
+    # Extract the years available
+    # Needed to make one change here - orig. tried to get years from df_ssp obj.
+    # like this: sat_years = df_ssp['year'][()], didn't work. related to which fair file it expected?
+    sat_years = ssp_folder["years"][()]
+    
+    # Which indices align with the reference and trim years
+    refyear_start_idx = np.flatnonzero(sat_years == refyear_start)[0]
+    refyear_end_idx = np.flatnonzero(sat_years == refyear_end)[0]  # + 1
+    year_start_idx = np.flatnonzero(sat_years == year_start)[0]
+    year_end_idx = np.flatnonzero(sat_years == year_end)[0] + 1
+
+    # Normalize and crop temperature series
+    Time = np.arange(year_start, year_end + 1)
+    SATave = np.mean(sat_ssp[refyear_start_idx:refyear_end_idx, :], axis=0)
+    SAT = sat_ssp[year_start_idx:year_end_idx, :] - SATave
+    # Close the h5 file
+    df_ssp.close()
+
+    return (SAT, Time, nens)
+
+
+def pickScenario(climate_data_file, scenario, rng):
+    """This function picks which samples to draw from (high v. low).
+    It calls GetSATData() which returns a normalized SAT and time array and number of ensemble members.
+    It then finds integrated SAT over 2000-2099, converts integrated temp into a normalized variable between low and high scenarios.
+    It then returns a boolean array 'useHigh' which indicates which samples to draw from high scenario.
+
+    """
+
+    SAT, Time, NumTensemble = GetSATData(climate_data_file, scenario)
```

Comment:
```suggestion
    SAT, Time, _ = GetSATData(climate_data_file, scenario)
```

Confirm this, but I think this `NumTensemble` might never gets used.